### DISCUSSION TITLE
Voting

### DISCUSSION THREAD

**Discussion by theshamkhi** [theshamkhi/GearBox_PC#9] [Polls]
**Option 1:** ![image](https://github.com/user-attachments/assets/f34089ff-a51d-49c9-9f8a-dac5700f6942)
**Option 2:** ![image](https://github.com/user-attachments/assets/195455a9-4d3f-495f-a5ad-c2f1946950dc)
**Option 3:** ![image](https://github.com/user-attachments/assets/fc338c46-7a5b-485b-a9ec-4f0278006754)
**Option 4:** ![image](https://github.com/user-attachments/assets/3a11ae21-e4a8-4000-b901-3874769fd974)

**Comment by OussamaBenoujja** [comment]
dude i already made my site themed with the white navbar and white footer there is no way i would want it changed especially since there is one day left for the deadline there a more critical features i gotta focus on right now.

**Reply by theshamkhi** [reply]
I understand but the others didn't agree with white so we had to vote

**Comment by OumaymaBrd** [comment]
Concernant mon avis, je privilégie l'option 2, car le fond de notre plateforme est blanc. Il serait préférable d'utiliser d'autres couleurs pour la barre de navigation et le pied de page
